modules/apps/object/object-test/src/testFunctional/tests/ObjectNotifications.testcase

```diff
@@ -663,13 +663,35 @@ definition {
 		}
 	}
 
-	@description = "LPS-171625 - Verify that after updating a system object entry, triggered email notifications using Object Author and Current User terms are filled correctly for the To, From Address, From Name, CC, BCC and Template Body fields"
+	@description = "LPS-171625 - Verify that after updating a custom object entry, triggered email notifications using Object Author and Current User terms are filled correctly for the To, From Address, From Name, CC, BCC and Template Body fields"
 	@priority = 5
-	test CanUseObjectAuthorAndCurrentUserTermsInEmailNotificationOnSystemObject {
+	test CanUseObjectAuthorAndCurrentUserTermsInEmailNotificationOnCustomObject {
 		property custom.properties = "feature.flag.LPS-171625=true";
 		property portal.acceptance = "true";
 
-		task ("Given an email notification template with the applicable fields filled with the Author and Current User terms") {
+		task ("Given create a Custom Object and add a field") {
```

Comment:
hey @JhossephG I think it could be better if you use something like: "Given a custom object, a text field and an object entry  created"